Windows/ppsspp.rc

```diff
@@ -304,6 +304,7 @@ BEGIN
     CONTROL         "Normal",IDC_MODENORMAL,"Button",BS_AUTORADIOBUTTON | WS_GROUP,248,9,40,9
     CONTROL         "Symbols",IDC_MODESYMBOLS,"Button",BS_AUTORADIOBUTTON,291,9,43,8
     GROUPBOX        "Mode",IDC_STATIC,241,0,104,22
+	AUTOCHECKBOX  "Show Offsets",IDC_SHOWOFFSETS,365,9,55,8

```

Comment:
Wrong type of tabs

Reply:
Oops, I missed that. I'll change it to four spaces like the rest.